frontend/src/static/js/components/webstatus-line-chart-tabbed-panel.ts

```diff
@@ -57,6 +57,23 @@ export abstract class WebstatusLineChartTabbedPanel<
     this.resetPointSelectedTask();
   }
 
+  getTabTooltip(tab: string): TemplateResult {
+    if (tab === 'Mobile') {
+      return html` <sl-tooltip
+        content="${'Collection of mobile test results is a work in progress.'}"
```

Comment:
I would mention results for `stable` channel somewhere here since we do have experimental. Here are some options:
- Work is underway to enable stable channel mobile test results
- Stable channel mobile test results are under active development and will expand as data becomes available
- Stable channel mobile test results: Work in progress

Reply:
Optionally, you can link https://github.com/web-platform-tests/interop-mobile-testing but not required

Reply:
Agreed. I've chosen your 1st option as the wording. 🙂 

> Optionally, you can link https://github.com/web-platform-tests/interop-mobile-testing but not required

I would normally think this is a good idea, but it is difficult to implement with the current use of a tooltip. 😕